specification_document-developments/2_0-Metabolomics-Draft/mzTab_2_0-M_mapping.xml

```diff
@@ -0,0 +1,171 @@
+-->
+<CvMapping xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.psidev.info/files/validator/CvMapping.xsd" modelName="mzTab_m_swagger.yml" modelURI="https://raw.githubusercontent.com/nilshoffmann/jmzTab-m/master/api/src/main/resources/mzTab_m_swagger.yml" modelVersion="2.0.0">
+
+    <CvReferenceList>
+      <CvReference cvName="PSI-MS" cvIdentifier="MS"/>
+      <CvReference cvName="DOID" cvIdentifier="DOID"/>
+      <CvReference cvName="CL" cvIdentifier="CL"/>
+      <CvReference cvName="BTO" cvIdentifier="BTO"/>
+      <CvReference cvName="NCBITaxon" cvIdentifier="NCBITaxon"/>
+<!--      <CvReference cvName="NEWT" cvIdentifier="NEWT"/>
+      <CvReference cvName="GO" cvIdentifier="GO"/>
+      <CvReference cvName="PATO" cvIdentifier="PATO"/>
+      <CvReference cvName="UO" cvIdentifier="UO"/>
+      <CvReference cvName="SEP" cvIdentifier="SEP"/>-->
+      <CvReference cvName="XLMOD" cvIdentifier="XLMOD"/>
+      <CvReference cvName="PRIDE" cvIdentifier="PRIDE"/>
+    </CvReferenceList>
+
+    <CvMappingRuleList>
+
+        <!-- run -->
+        <CvMappingRule id="msrun_format_may" cvElementPath="/metadata/msRun/@format" requirementLevel="MAY"  scopePath="/metadata/msRun" cvTermsCombinationLogic="AND">
+          <CvTerm termAccession="MS:1000560" useTerm="false" termName="mass spectrometer file format" isRepeatable="false" allowChildren="true" cvIdentifierRef="MS"></CvTerm>
+        </CvMappingRule> 
+        
+        <CvMappingRule id="msrun_id_format_may" cvElementPath="/metadata/msRun/@idformat" requirementLevel="MAY"  scopePath="/metadata/msRun" cvTermsCombinationLogic="AND">
+          <CvTerm termAccession="MS:1000767" useTerm="false" termName="native spectrum identifier format" isRepeatable="false" allowChildren="true" cvIdentifierRef="MS"></CvTerm>
+        </CvMappingRule> 
+        
+        <CvMappingRule id="msrun_fragmentation_method_may" cvElementPath="/metadata/msRun/@fragmentationMethod" requirementLevel="MAY"  scopePath="/metadata/msRun" cvTermsCombinationLogic="OR">
+          <CvTerm termAccession="MS:1000044" useTerm="false" termName="dissociation method" isRepeatable="true" allowChildren="true" cvIdentifierRef="MS"></CvTerm>
+        </CvMappingRule> 
+        
+        <CvMappingRule id="msrun_scan_polarity_must" cvElementPath="/metadata/msRun/@scanPolarity" requirementLevel="MUST"  scopePath="/metadata/msRun" cvTermsCombinationLogic="OR">
+          <CvTerm termAccession="MS:1000129" useTerm="true" termName="negative scan" isRepeatable="false" allowChildren="false" cvIdentifierRef="MS"></CvTerm>
+          <CvTerm termAccession="MS:1000130" useTerm="true" termName="positive scan" isRepeatable="false" allowChildren="false" cvIdentifierRef="MS"></CvTerm>
+        </CvMappingRule> 
```

Comment:
What about GC/MS ? no scanPolarity attribute in first place ?

Reply:
GC-MS supports positive ionization via electron ionization (EI) while it also supports negative ionization when negative chemical ionization (NCI) is being used.

Reply:
For LECO systems, this will usually be positive mode, but Shimadzu also offers NCI systems.